Explicitly include sys/time.h for timeval on UNIX. Required on non-glibc
systems.

--- a/src/ccutil/ocrclass.h
+++ b/src/ccutil/ocrclass.h
@@ -31,6 +31,8 @@
 #ifdef _WIN32
 #include <winsock2.h> // for timeval
+#else
+#include <sys/time.h> // for timeval
 #endif
 
 /**********************************************************************
  * EANYCODE_CHAR
